device/arista/x86_64-arista_7170_64c/Arista-7170-64C/buffers_defaults_t0.j2

```diff
@@ -10,37 +14,70 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            "size": "{{ ingress_lossless_pool_size }}",
             "type": "ingress",
-            "mode": "dynamic",
-            "xoff": "7827456"
+            "mode": "dynamic"
         },
-        "egress_lossy_pool": {
-            "size": "26663272",
-            "type": "egress",
+        "ingress_lossy_pool": {
+            "size": "{{ ingress_lossy_pool_size }}",
+            "type": "ingress",
             "mode": "dynamic"
         },
         "egress_lossless_pool": {
-            "size": "42349632",
+            "size": "{{ egress_lossless_pool_size }}",
             "type": "egress",
-            "mode": "static"
+            "mode": "dynamic"
+        },
+        "egress_lossy_pool": {
+            "size": "{{ egress_lossy_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
         }
     },
     "BUFFER_PROFILE": {
-        "ingress_lossy_profile": {
+        "ingress_lossless_profile": {
             "pool":"[BUFFER_POOL|ingress_lossless_pool]",
-            "size":"0",
-            "static_th":"11075584"
+            "size":"4096",
+            "dynamic_th":"0"
+        },
+        "ingress_lossy_profile": {
+            "pool":"[BUFFER_POOL|ingress_lossy_pool]",
+            "size":"4096",
+            "dynamic_th":"3"
         },
         "egress_lossless_profile": {
             "pool":"[BUFFER_POOL|egress_lossless_pool]",
-            "size":"0",
-            "static_th":"10587408"
+            "size":"4096",
+            "dynamic_th":"7"
```

Comment:
we should avoid any egress packet drop for lossless traffic. previously configuration was to avoid any packet drop at egress. I do not think current configuration can guarantee no packet drop at egress for lossless traffic.